src/clj/athens/self_hosted/web/presence.clj

```diff
@@ -13,7 +14,7 @@
     (swap! max-id inc)))
 
 
-(defonce all-presence (ref []))
+(defonce all-presence (atom {}))
```

Comment:
You don't like `dosync` blocks?
![Rick wink](https://media.giphy.com/media/cjDax1AZDFWO7S60MM/giphy.gif)

Reply:
Just more used to atom semantics really.